salt/modules/lxc.py

```diff
@@ -989,6 +1045,10 @@ def init(name,
     if users is None:
         users = []
     dusers = ['root']
+    # think to touch the default user generated by default ubuntu template which
+    # has a really unsecure password too ...
+    if __grains__['os'] in ['Ubuntu'] and 'ubuntu' not in users:
+        dusers.append('ubuntu')
```

Comment:
No, this code was removed for a reason. When the host machine is Ubuntu and a non-Ubuntu container is created using the **download** template (or if a container is installed from a rootfs image), the minion will attempt to set a password for a nonexistant ``ubuntu`` user.

A better solution for this, if you do not want to pass a ``users`` param, would be to add ``users`` as a supported param for container profiles, and retrieve it via the ``select()`` function.